src/Simulation/Native/src/external/fused.hpp

```diff
@@ -41,7 +41,14 @@ class Fused
       fusedgates = Fusion();
     }
 
+    Fusion get_fusedgates() const& {
```

Comment:
most people would be more familiar with signature like `const Fusion& get_fusedgates() const`, the first const referring to the returned type `Fusion&` and the second stating that the function doesn't change the object it's invoked on. I'm actually not sure what the signature with `const&` after function name means and I wouldn't be looking forward to reading the C++ standard on this (I'm surprised it compiles) :D